purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -0,0 +1,206 @@
+package com.revenuecat.purchases.paywalls
+
+import java.net.URL
+
+/**
+ * Represents the data required to display a paywall using the `RevenueCatUI` library.
+ * This data can be created and configured in the dashboard and then accessed from the `Offering/paywall`.
+ *
+ * @see [Paywalls Documentation](https://rev.cat/paywalls)
+ */
+data class PaywallData(
+    /**
+     * The type of template used to display this paywall.
+     */
+    val templateName: String,
+
+    /**
+     * Generic configuration for any paywall.
+     */
+    val config: Configuration,
+
+    /**
+     * The base remote URL where assets for this paywall are stored.
+     */
+    val assetBaseURL: URL,
+    internal val localization: Map<String, LocalizedConfiguration>,
+) {
+    /**
+     * Generic configuration for any paywall.
+     */
+    data class Configuration(
+        /**
+         * The list of package identifiers this paywall will display.
+         */
+        val packages: List<String>,
+
+        /**
+         * The package to be selected by default.
+         */
+        val defaultPackage: String? = null,
+
+        /**
+         * The images for this template.
+         */
+        val images: Images,
+
+        /**
+         * Whether the background image will be blurred (in templates with one).
+         */
+        val blurredBackgroundImage: Boolean = false,
+
+        /**
+         * Whether a restore purchases button should be displayed.
+         */
+        val displayRestorePurchases: Boolean = true,
+
+        /**
+         * If set, the paywall will display a terms of service link.
+         */
+        val termsOfServiceURL: URL? = null,
+
+        /**
+         * If set, the paywall will display a privacy policy link.
+         */
+        val privacyURL: URL? = null,
+
+        /**
+         * The set of colors used.
+         */
+        val colors: ColorInformation,
+    ) {
+        data class Images(
+            /**
+             * Image displayed as a header in a template.
+             */
+            val header: String? = null,
+
+            /**
+             * Image displayed as a background in a template.
+             */
+            val background: String? = null,
+
+            /**
+             * Image displayed as an app icon in a template.
+             */
+            val icon: String? = null,
```

Comment:
`= null` is so we have the empty constructor

Reply:
I believe that's the case yes https://github.com/RevenueCat/purchases-ios/blob/paywalls/Tests/UnitTests/Networking/Responses/Fixtures/Offerings.json#L75

Maybe @NachoSoto can confirm